styles/style.css

```diff
@@ -0,0 +1,120 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+
+
+*,
+html {
+    scroll-behavior: smooth !important;
```

Comment:
Style: The important tag should be avoided wherever possible.  

See MDN: https://developer.mozilla.org/en-US/docs/Web/CSS/important#best_practices

Reply:
One thing that's worth noting, smooth scrolling can give some people motion sickness (including me).